site/layouts/_default/docs.html

```diff
@@ -30,5 +30,9 @@ <h1 class="bd-title" id="content">{{ .Title | markdownify }}</h1>
     </div>
 
     {{ partial "scripts" . }}
+
+    {{ range .Page.Params.external_js }}
+      <script src="{{ .src }}" integrity="{{ .integrity }}" crossorigin="anonymous"></script>
```

Comment:
I'd check for `.integrity` in case we don't want a script to have it. (with `{{ with .integrity }} integrity="{{ . }}"{{ end }}` or something similar if this doesn't work due to the scope of `.`).

Reply:
@XhmikosR, could you check if this is the way to go, not sure how the scope of variables works in Hugo:
https://github.com/twbs/bootstrap/blob/6cc2142f4b69e0c1609325e189cff3919b96f43c/site/layouts/_default/examples.html#L57-L61

Reply:
When you are done moving this to an example, I will be able to try it and let you know.